applications/poisson/gaussian/tests/gaussian.output

```diff
@@ -59,6 +59,7 @@ Solver:
       Smoother sweeps:                       1
       Number of cycles:                      1
       Smoother type:                         ILU
+      AMG operator type:                     Laplace
```

Comment:
If we consider `Laplace` the correct value for the `ExaDG::Poisson` module and `Elasticity` the correct value for the `ExaDG::Structure` module (developments happening actually in Trilinos/deal.II), there is no need print this parameter in my opinion. I rather tend to add asserts deep in the code in ExaDG making sure that the preconditioner implemenetations receive a certain, expected  value. (If one wants to change this parameter for experimentation, one would have to go to this place in ExaDG with the assert, comment the assert, and overwrite the value. I consider this justified right now because I consider the development of these preconditioners settings and optimal parameter values as developments happening outside ExaDG.)

Reply:
ok